docs/contributor/adr/0010-support-for-enable-prometheusMerge-configuration.md

```diff
@@ -0,0 +1,70 @@
+# Support for Enable PrometheusMerge Configuration
+
+## Status
+Approved
+
+## Context
+There is a need to support the configuration of enabling `prometheusMerge` to be available in the Istio CR. The background behind this need is explained in [this issue](https://github.com/kyma-project/telemetry-manager/issues/1468) and the impact of this feature being enabled was discussed in an [ADR](https://github.com/kyma-project/telemetry-manager/blob/main/docs/contributor/arch/015-impact-of-istio-prometheus-merge-on-metric-pipelines.md) created in the Telemetry Manager repository.
+
+## Considerations
+Based on the [original ADR](https://github.com/kyma-project/telemetry-manager/blob/main/docs/contributor/arch/015-impact-of-istio-prometheus-merge-on-metric-pipelines.md), the plan was to originally enable the `prometheusMerge` feature by default. But this has caused problems for users deploying prometheus with a scrape loop sidecar. More details can be found in [this PR](https://github.com/kyma-project/istio/pull/1184).
+
+
+## Decision
+The enablement of the `prometheusMerge` feature needs to be a configurable option for the users. By default it will be set to `false`, but users will then need to actively set it to `true` and understand the effects of the feature.
```

Comment:
```suggestion
Enabling the `prometheusMerge` feature must be a configurable option for the users. By default, it will be set to `false`, and users must actively set it to `true` and understand the effects of the feature.
```